src/components/tabs/ConfigurationTab.vue

```diff
@@ -0,0 +1,1117 @@
+                                        </td>
+                                    </tr>
+                                </tbody>
+                            </table>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="content_toolbar toolbar_fixed_bottom" style="position: fixed">
+            <div class="btn save_btn">
+                <a class="save" href="#" @click.prevent="saveConfig">{{ $t("configurationButtonSave") }}</a>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+import { defineComponent, ref, reactive, onMounted, computed, nextTick, watch } from "vue";
+import GUI from "../../js/gui";
+import FC from "../../js/fc";
+import MSP from "../../js/msp";
+import MSPCodes from "../../js/msp/MSPCodes";
+import { mspHelper } from "../../js/msp/MSPHelper.js";
+import { reinitializeConnection } from "../../js/serial_backend";
+import { gui_log } from "../../js/gui_log";
+import { i18n } from "../../js/localization";
+import { sensorTypes } from "../../js/sensor_types"; // Import for dropdown lists
+import { have_sensor } from "../../js/sensor_helpers";
+import semver from "semver";
+import { API_VERSION_1_45, API_VERSION_1_46, API_VERSION_1_47 } from "../../js/data_storage";
+import { bit_check, bit_set, bit_clear } from "../../js/bit";
+import { updateTabList } from "../../js/utils/updateTabList";
+
+export default defineComponent({
+    name: "ConfigurationTab",
+    setup() {
+        // Reactive State
+        const pidAdvancedConfig = reactive({
+            gyro_sync_denom: 1,
+            pid_process_denom: 1,
+        });
+
+        const sensorConfig = reactive({
+            acc_hardware: 0,
+            baro_hardware: 0,
+            mag_hardware: 0,
+            sonar_hardware: 0,
+            opticalflow_hardware: 0,
+        });
+
+        const boardAlignment = reactive({
+            roll: 0,
+            pitch: 0,
+            yaw: 0,
+        });
+
+        const fpvCamAngleDegrees = ref(0);
+
+        const craftName = ref("");
+        const pilotName = ref("");
+        const showPilotName = ref(false);
+
+        const armingConfig = reactive({
+            small_angle: 25,
+            gyro_cal_on_first_arm_bool: false,
+            auto_disarm_delay: 0,
+        });
+
+        const accelTrims = reactive({
+            roll: 0,
+            pitch: 0,
+        });
+
+        const sensorAlignment = reactive({
+            gyro_to_use: 0,
+            gyro_1_align: 0,
+            gyro_2_align: 0,
+            align_mag: 0,
+        });
+
+        const magDeclination = ref(0);
+        const showGyroCalOnFirstArm = ref(false);
+        const showAutoDisarmDelay = ref(false);
+        const hasSecondGyro = ref(false);
+        const hasDualGyros = ref(false);
+        const showGyroToUse = computed(() => {
+            return semver.lt(FC.CONFIG.apiVersion, API_VERSION_1_47)
```

Comment:
```suggestion
            return semver.lt(FC.CONFIG.apiVersion, API_VERSION_1_47);
```

Reply:
We also need to check

<img width="1135" height="126" alt="image" src="https://github.com/user-attachments/assets/afb08ba0-600c-4d4f-b4e6-52f0a1b3bd92" />